.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,120 @@
+name: Bug report
+description: General bugs, program crashes, broken files, wrong values, etc...
+title: "[BUG]"
```

Comment:
Nit: Add trailing space, so the user can start typing right away. Since we probably want `[BUG] Foobar` (spaced) and not `[BUG]Foobar` formatting.